website/views/manage_carousel.py

```diff
@@ -0,0 +1,120 @@
+from django.shortcuts import render, redirect
```

Comment:
It is mainly to order each line, not necessarily the elements of each line. So, more like your second case.

Also, officially (https://www.python.org/dev/peps/pep-0008/#imports), if you look at the third point, they should be separated in 3 blocks: system / basic python imports first, then the third-party imports (like Django), and then the imports from the current project.